tests/lib/tools/boot-state

```diff
@@ -107,7 +107,7 @@ bootenv_set() {
     if run_grub_editenv set "$var=$value"; then
         return
     elif [ -s "$grubenv_file" ]; then
-        sed -i "/^$var=/d" "$grubenv_file"
+        sed --follow-symlinks -i "/^$var=/d" "$grubenv_file"
```

Comment:
Is this change related to this PR?

Reply:
yes, it happened that otherwise it was breaking the symlinks. After sed (removing a line), the file was not a symlink anymore, it was a regular file.